<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scrape tokoalvabet.com</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>HOME URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>COLUMNS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>FILENAME</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>461</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="homeURLPTE">
        <property name="plainText">
         <string>https://www.tokoalvabet.com</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="columnsPTE">
        <property name="plainText">
         <string>['Judul', 'Harga', 'Penulis', 'Genre', 'Cetakan', 'Bulan','Tahun','Lebar', 'Panjang','Tebal','Berat', 'Format', 'ISBN', 'Stock', 'Sinopsis', 'Tentang_penulis','Foto Produk1', 'Foto Produk2',]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="filenamePTE">
        <property name="plainText">
         <string>tokoalvabet.csv</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>541</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextEdit" name="logTE"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="scrapeBtn">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Scrape!</string>
     </property>
    </widget>
    <widget class="QLabel" name="timestartLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Time start :</string>
     </property>
    </widget>
    <widget class="QLabel" name="timefinishLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>500</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Time finish :</string>
     </property>
    </widget>
    <widget class="QLabel" name="numrowsLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of rows :</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
